ci-operator/config/openshift/project-request-limit/openshift-project-request-limit-master.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: "4.5"
+    namespace: ocp
+    tag: base
+binary_build_commands: make build --warn-undefined-variables
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- dockerfile_path: images/ci/Dockerfile
+  from: base
```

Comment:
missing injection of builder image https://github.com/openshift/release/blob/6c6b25af3a4b20f6a45481b2a8b627773d347c9f/ci-operator/config/openshift/cluster-kube-apiserver-operator/openshift-cluster-kube-apiserver-operator-master.yaml#L17-L21

also this isn't rhel dockerfile

Reply:
@tnozicka added `injection of builder image`, although not sure why we need it. Is there any document that clarifies why we need it and what it does?

Reply:
I think you can reference like so:
```
  bin:
    as:
    - builder
```